resources/web/template.html

```diff
@@ -82,32 +82,46 @@
         <div class="content-wrapper">
 
           <section id="guide" <!-- DOCS LANG -->>
-            <div class="container">
+            <div class="content-container">
               <div class="row">
-
-                <div class="col-12 order-2 col-sm-3 order-sm-1 col-md-3 order-md-1" id="left_col">
-                  <div id="rtpcontainer" style="display: block;">
-                    <div class="mktg-promo">
-                      <h3>Most Popular</h3>
-                      <ul class="icons">
-                        <li class="icon-elasticsearch-white"><a href="https://www.elastic.co/webinars/getting-started-elasticsearch?baymax=default&elektra=docs&storm=top-video">Get Started with Elasticsearch: Video</a></li>
-                        <li class="icon-kibana-white"><a href="https://www.elastic.co/webinars/getting-started-kibana?baymax=default&elektra=docs&storm=top-video">Intro to Kibana: Video</a></li>
-                        <li class="icon-logstash-white"><a href="https://www.elastic.co/webinars/introduction-elk-stack?baymax=default&elektra=docs&storm=top-video">ELK for Logs & Metrics: Video</a></li>
-                      </ul>
-                    </div>
-                  </div>
+                <div class="col-12 order-2 col-sm-3 order-sm-1 col-md-3 order-md-1 col-lg-3 sticky-top h-almost-full" id="left_col">
+                  <!-- Can we pull this in? -->
+                  <div class="book-title">{Book name}</div>
```

Comment:
If we can, it wouldn't be done here. We'd need to prepend the book name to the TOC when it's built.

Reply:
I'll look at how we access the book title on index pages. Maybe we can use the same logic.

Reply:
On second thought, I think this is a non-issue. The title of the book and version selector are omitted when building locally. When doing a full build, the title and version appear at the top of the TOC:

<img width="378" alt="Screen Shot 2022-07-20 at 1 23 55 PM" src="https://user-images.githubusercontent.com/5618806/180074849-09a13928-9091-402d-869e-ea01188dce43.png">
.

Reply:
First pass looks like: 

<img width="1568" alt="Screen Shot 2022-07-21 at 2 24 35 PM" src="https://user-images.githubusercontent.com/10479155/180303255-a31b80b5-32e2-4138-8f7d-f811ff5fa528.png">